Sources/TuistLoader/Models+ManifestMappers/Target+ManifestMapper.swift

```diff
@@ -195,9 +199,21 @@ extension TuistGraph.Target {
     }
 }
 
-extension Array where Element == TuistGraph.ResourceFileElement {
-    mutating func remove(path: AbsolutePath) {
-        guard let index = firstIndex(of: TuistGraph.ResourceFileElement(path: path)) else { return }
-        remove(at: index)
+enum ResourceFileElementError: FatalError {
+    case globsNotAllowed
```

Comment:
Instead of creating a different error, we can add this to `TargetManifestMapperError`